app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -38,6 +38,16 @@ examples:
       suggestion_link_text: "You can now apply for a permit online."
       suggestion_link_url: "/permit"
 
+  open_suggestion_link_in_new_tab:
+    description: |
+      For example if you wanted to drive users to another online task, you don't want to completely hijack
+      their original flow on GOV.UK. Adding the option to open links in a new tab helps to address this.
+      Link text *must* tell the user that the link opens in a new tab.
```

Comment:
I wonder if we can add that "(opens in a new tab)" text automatically if `new_tab` is specified.

Reply:
Love this suggestion.

Reply:
[Done](https://github.com/alphagov/govuk_publishing_components/pull/2465/commits/9f13ad48ccf450c3b81df321f289c19d0af44128)